apps/server/src/modules/tldraw/controller/api-test/tldraw.ws.api.spec.ts

```diff
@@ -97,32 +114,145 @@ describe('WebSocketController (WsAdapter)', () => {
 			expect(handleConnectionSpy).toHaveBeenCalled();
 			expect(handleConnectionSpy).toHaveBeenCalledTimes(2);
 
+			handleConnectionSpy.mockRestore();
 			ws.close();
 			ws2.close();
 		});
 	});
 
-	describe('when tldraw is not correctly setup', () => {
-		const setup = async () => {
-			const handleConnectionSpy = jest.spyOn(gateway, 'handleConnection');
+	describe('when checking cookie', () => {
+		const setup = () => {
+			const closeClientSpy = jest.spyOn(gateway, 'closeClient');
```

Comment:
closeClient is your own method but you should mock and spy the actual client: WebSocket